checker/jdk/nullness/src/java/lang/Class.java

```diff
@@ -6,8 +6,13 @@
 import org.checkerframework.checker.nullness.qual.PolyNull;
 import org.checkerframework.dataflow.qual.Pure;
 import org.checkerframework.dataflow.qual.SideEffectFree;
+import org.checkerframework.framework.qual.Covariant;
 
-public final class Class<T> extends Object implements java.io.Serializable, java.lang.reflect.GenericDeclaration, java.lang.reflect.Type, java.lang.reflect.AnnotatedElement {
+// The type annotation on an instantiation of Class is meaningless.
+// Class<@NonNull String> and Class<@Nullable String> have the same
+// meaning, but unfortunately are not interchangeable.
+@Covariant(0)
```

Comment:
Just to be absolutely clear, maybe expand on the `interchangeable` and say that the `@Covariant` allows one direction, but not the other.